hack/sync-ec-cli-tasks.sh

```diff
@@ -25,29 +25,31 @@ set -o pipefail
 set -o nounset
 
 EC_CLI_REPO_PATH="${1}"
+RELEASE_PREFIX="release-v"
 
-cp -r "${EC_CLI_REPO_PATH}/tasks" .
+collect_remotes() {
+  echo "$(git branch --remote --format '%(refname:lstrip=-1)' |grep ${RELEASE_PREFIX} | sort -u)"
```

Comment:
I think this could work as well:
```suggestion
  git branch --remote --format '%(refname:lstrip=-1)' --sort=refname --list 'origin/release-v*'
```

The `origin/release-v*` will list only the branches from the `origin` remote, sorting just for kicks, I think `sort -u` was used to exclude the equally named branches from different remotes...

Reply:
That's good. thanks.